community/sustainability_effort.md

```diff
@@ -0,0 +1,80 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) and in return have their PRs prioritized for review.
+
+Help is also needed for triaging new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels.
+
+## Commitment
+
+The Argo project currently has 4 designated [roles](https://github.com/argoproj/argoproj/blob/main/community/membership.md):
+- Member
+- Reviewer
+- Approver
+- Lead
+
+For those who focus on just one area of the code (such as UI), the `Reviewer` and `Approver` roles can optionally be scoped to just that area.
+
+Any person in a designated role (or even those who are not yet Members) can sign up to participate in this Incentive System.
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member plus those not yet Member: should focus primarily on authoring PRs, should average a minimum of 8 hours per week; can also assist with triaging bugs or optionally reviewing PRs
```

Comment:
```suggestion
- Member and below: should focus primarily on authoring PRs, averaging a minimum of 8 hours per week. Can also assist with triaging bugs or reviewing PRs.
```

- fix run-on sentence
- shortened to "Member and below" as we do regularly use language like "Approver+" already